supervisor/resolution/notify.py

```diff
@@ -48,7 +48,7 @@ async def issue_notifications(self):
                     messages.append(
                         {
                             "title": "Security notification",
-                            "message": "The Supervisor detected that this version of Home Assistant is insecure. [Update as soon as possible.](/hassio/dashboard)\n\nFor more information see the [Security bulletin](https://www.home-assistant.io/blog/2021/01/14/security-bulletin/).",
+                            "message": "The Supervisor detected that this version of Home Assistant could be insecure in combination with custom integrations. [Update as soon as possible.](/hassio/dashboard)\n\nFor more information see the [Security bulletin](https://www.home-assistant.io/blog/2021/01/14/security-bulletin/).",
```

Comment:
The Supervisor detected that you are running a Home Assistant Core version that could be insecure in combination with custom integrations.